met-api/src/met_api/config.py

```diff
@@ -92,7 +92,13 @@ class _Config():  # pylint: disable=too-few-public-methods
     SURVEY_PATH = os.getenv('SURVEY_PATH', '/survey/submit/{survey_id}/{token}')
     ENGAGEMENT_PATH = os.getenv('ENGAGEMENT_PATH', '/engagement/view/{engagement_id}')
     SITE_URL = os.getenv('SITE_URL')
+
     VERIFICATION_EMAIL_SUBJECT = os.getenv('VERIFICATION_EMAIL_SUBJECT', '{engagement_name} - Survey link')
+    VERIFICATION_EMAIL_TEMPLATE_ID = os.getenv('VERIFICATION_EMAIL_TEMPLATE_ID')
+
+    ENGAGEMENT_CLOSEOUT_EMAIL_TEMPLATE_ID = os.getenv('ENGAGEMENT_CLOSEOUT_EMAIL_TEMPLATE_ID')
+    ENGAGEMENT_CLOSEOUT_EMAIL_SUBJECT = \
+        os.getenv('ENGAGEMENT_CLOSEOUT_EMAIL_SUBJECT', '{engagement_name} - What we heard')
```

Comment:
Yep, i'm moving the clouseout env variables to the cron job and keeping the other ones here, there's a slight duplication but it should be okay since they are different services